### PR TITLE
Docs: update geomap docs to include Base layer in the four Data layer types list

### LINKED ISSUE #44977: Docs feedback: /visualizations/geomap.md
Seems to be something missing here. 
image

### PR DESCRIPTION
**Pull request by Seyaji**
This PR updates the docs to include Base layer in the four Data layer types list,
it also gives a further explanation about the availability of Base Layers  in the Data layer panel option and the Base Layer panel option

Fixes #44977

### CODE REVIEW

File: docs/sources/visualizations/geomap.md [hunk 1]
```diff
@@ -180,4 +181,4 @@ default_baselayer_config = `{
 }`
 ```
 
-`enable_custom_baselayers` allows you to enable or disable custom open source base maps that are already implemented. The default is `true`.
+`enable_custom_baselayers` allows you too enable or disable custom open source base maps that are already implemented. The default is `true`.
```
Comment (daniellee) [inline docs/sources/visualizations/geomap.md:184]: This looks incorrect. Should be `to` and not `too` as it allows you **to** enable or disable.

Comment (Seyaji) [reply docs/sources/visualizations/geomap.md:184]: 😅 I confuse myself sometimes! Thanks!

**Comment by user10x** [comment]
Just following this issue as a way to contribute to others, seems like you have to backport this to other release branches among other checks?

https://github.com/grafana/grafana/blob/main/contribute/merge-pull-request.md

File: docs/sources/visualizations/geomap.md [hunk 2]
```diff
@@ -115,7 +116,7 @@ The GeoJSON layer allows you to select and load a static GeoJSON file from the f
 
 ## Base layer
 
-The base layer loads in a blank world map from the tile server to the Grafana panel. Several base layer options are available each with specific configuration options to style the base map. The default base layer is CartoDB base map. Custom default base layers can be defined in the `.ini` configuration file.
+The base layer loads in a blank world map from the tile server to the Grafana panel. Several base layer options are available each with specific configuration options to style the base map. Base layers can be added to the Data layer stack using layer controls in the Data layer panel option, or configured separately using the Base layer panel option. Adding multiple base layers to the Data layer stack allows you to overlay them, you can use the opacity controls to balance the overlay mix. The default base layer is CartoDB base map. Custom default base layers can be defined in the `.ini` configuration file.
```
Comment (gguillotte-grafana) [inline docs/sources/visualizations/geomap.md:119]:
```suggestion
The base layer loads in a blank world map from the tile server to the Grafana panel. Several base layer options are available each with specific configuration options to style the base map. Base layers can be added to the Data layer stack using layer controls in the Data layer panel option, or configured separately using the Base layer panel option. To overlay layers, add multiple base layers to the Data layer stack, and use the opacity controls to balance the overlay mix. The default base layer is CartoDB base map. Custom default base layers can be defined in the `.ini` configuration file.
```

**Review by gguillotte-grafana — APPROVED** [review]
Added a suggested rewording, but this shouldn't block merging.

**Comment by Seyaji** [comment]
@gguillotte-grafana Thanks, I have committed the suggestion 🙂

**Comment by drew08t** [comment]
Appreciate the update! This has been captured in https://github.com/grafana/grafana/pull/53552 and this PR can be closed.

**Pull request closed without merging**